README.md

```diff
@@ -671,6 +672,12 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Logto](https://logto.io/) - Develop, secure, and manage user identities of your product - for both authentication and authorization. Free for up to 5,000 MAUs with open-source self-hosted option available.
 
 
+**[⬆️ Back to Top](#table-of-contents)**
+
+## Mobile App Distribution and Feedback
+
+* [TestApp.io](https://testapp.io) - Your go-to platform for making sure your mobile apps work like they should. Catering to both Android and iOS, whether it's a game, shopping app, or any other mobile experience, we simplify the process. The free plan offers support for 1 app (Android & iOS), basic analytics, internal distribution, unlimited external testers, and feedback collection.
```

Comment:
Thanks, afraid we dont add single entry categories.  EIther move other entries that match this new category or add yours to an existing category.

The description is also a bit long, try remove some marketing speak and focus on the actual features.

Reply:
@ripienaar noted. I've reduced the description and added similar platforms to the category.